libs/application/templates/parental-leave/src/lib/dataSchema.ts

```diff
@@ -72,8 +72,11 @@ export const dataSchema = z.object({
         .optional(),
     })
     .optional(),
-  singlePeriod: z.enum([YES, NO]),
-  confirmLeaveDuration: z.enum(['duration', 'specificDate']),
+  transferRights: z.enum([
+    TransferRightsOption.REQUEST,
+    TransferRightsOption.GIVE,
+    TransferRightsOption.NONE,
+  ]),
```

Comment:
Should this replace `requestRights.isRequestingRights` and `giveRights.isGivingRights`?

Reply:
Yes, eventually that would be ideal.